docs/most-common-issues.md

```diff
@@ -0,0 +1,14 @@
+Before you do anything else try to remove the project again from your computer and clone it again
+
+```bash
+rm -rf [yourpath]/debugger.html
+git clone https://github.com/devtools-html/debugger.html
+cd debugger.html
+yarn install
+yarn start
+```
+
+Don't forget to ask in our [slack room][slack] or fill an [issue][yarn-issue] if you get stuck.
+We are always happy to help.
+
+[slack]:https://devtools-html-slack.herokuapp.com/
```

Comment:
lets mention that you should check your version of node as well.

Also, if that doesn't work.... use nvm to switch to a clean version of node w/ no global dependencies